### ISSUE TITLE
[BUG] Keyboard hiding animation

### ISSUE THREAD

**Issue by bonsoirelliot** [surfstudio/flutter-bottom-sheet#28]
## Describe the bug

Hello! I have a TextField inside FlexibleBottomSheet. When the keyboard is hiding there is a hole in the bottom of FlexibleBottomSheet for some period of time. Looks like the animation of FlexibleBottomSheet doesn't keep up with the keyboard animation. 

```dart
`showFlexibleBottomSheet<void>(
                    minHeight: 0,
                    initHeight: 0.5,
                    maxHeight: 0.6,
                    context: context,
                    builder: (context, controller, i) {
                      return Scaffold(
                        resizeToAvoidBottomInset: false,
                        body: CustomScrollView(
                          controller: controller,
                          slivers: [
                            SliverList(
                              delegate: SliverChildListDelegate(
                                [
                                  Text(
                                    'text',
                                  ),
                                  TextField(),
                                ],
                              ),
                            ),
                          ],
                        ),
                      );
                    });`
```

## What is the expected behavior?

Hide keyboard without hole in bottomSheet. 
If paste TextField inside ModalBottomSheet, keyboard hiding animation is working right.

## Additional context

* On IOS keyboard hiding animation works fine.
  
* I hope gifs are supported
* ![alt text](https://github.com/bonsoirelliot/images/blob/main/ezgif-2-df4430363b.gif?raw=true)

* `flutter doctor -v` 
[√] Flutter (Channel stable, 2.5.3, on Microsoft Windows [Version 10.0.19043.1415], locale ru-RU)
    • Flutter version 2.5.3 at C:\flutter\flutter
    • Upstream repository https://github.com/flutter/flutter
    • Framework revision 18116933e7 (2 months ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at C:\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.63.2)
    • VS Code at C:\Users\Никита\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.29.0

[√] Connected device (3 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Chrome (web)            • chrome        • web-javascript • Google Chrome 96.0.4664.110
    • Edge (web)              • edge          • web-javascript • Microsoft Edge 94.0.992.38

• No issues found!

**Comment by MbIXjkee** [comment]
Thanks for this information. We will check this behavior.

**Comment by MbIXjkee** [comment]
Fixed in 2.1.0 version
